cmd/notary/main_test.go

```diff
@@ -369,3 +370,89 @@ func TestConfigFileOverridenByCmdLineFlags(t *testing.T) {
 	require.Len(t, m.gotten, 1)
 	require.Equal(t, m.gotten[0], "repo.root")
 }
+
+// the config can specify trust pinning settings for TOFUs, as well as pinned Certs or CA
+func TestConfigFileTrustPinning(t *testing.T) {
+	var err error
+
+	tempDir := tempDirWithConfig(t, `{
+        "trust_pinning": {
+            "disable_tofu": false
+         }
+	}`)
+	defer os.RemoveAll(tempDir)
+	commander := &notaryCommander{
+		getRetriever: func() passphrase.Retriever { return passphrase.ConstantRetriever("pass") },
+		configFile:   filepath.Join(tempDir, "config.json"),
+	}
+
+	// Check that tofu was set correctly
+	config, err := commander.parseConfig()
+	require.NoError(t, err)
+	require.Equal(t, false, config.GetBool("trust_pinning.disable_tofu"))
+	trustPin, err := getTrustPinning(config)
+	require.NoError(t, err)
+	require.Equal(t, false, trustPin.DisableTOFU)
+
+	tempDir = tempDirWithConfig(t, `{
+		"remote_server": {
+			"url": "%s"
+		},
+        "trust_pinning": {
```

Comment:
non-blocking nitpick:  I think this block got unindented :)  But it's just a string, so doesn't really matter